test/bigtest/tests/financial-transaction-report-test.js

```diff
@@ -0,0 +1,56 @@
+import {
+  beforeEach,
+  describe,
+  it,
+} from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import setupApplication from '../helpers/setup-application';
+import UsersInteractor from '../interactors/users';
+
+describe('Financial transaction detail report', () => {
+  const users = new UsersInteractor();
+
+  setupApplication();
+
+  describe('When patron has financial transaction detail report permissions', () => {
+    setupApplication({
+      hasAllPerms: false,
+      permissions: {
+        'module.users.enabled': true,
+        'ui-users.financialTransactionReport': true,
+      },
+    });
```

Comment:
I guess you can omit here `setupApplication`, because  by default it has `hasAllPerms: true`.